cargo-dist/templates/installer/installer.ps1.j2

```diff
@@ -55,6 +50,24 @@ $receipt = @"
 "@
 $receipt_home = "${env:LOCALAPPDATA}\{{ app_name }}"
 
+if ($env:{{ disable_update_env_var }}) {
+  $install_updater = $false
+} else {
+  $install_updater = $true
+}
+
+$NoModifyPath = $false
```

Comment:
format of this variable name is weird now but not a big deal?

Reply:
Marking as resolved since I've decided to keep the param for consistency with shell.